### PR TITLE
chore: updated sample config

### PR DESCRIPTION
**Pull request by 0xrajath**
**Motivation:**

We need to update the sample config to include the latest factory and token adapter logic addresses.
We also need to update the deployment info with the deployment commit hash.

**Modifications:**

* Updated sample config
* Updated deployment info

**Result:**

More accurate info.

### CODE REVIEW

File: build_info/token_voting_module_deployment_info.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 Deployed with:
+
```
Comment (0xrajath) [inline build_info/token_voting_module_deployment_info.md:2]: My prettier seems to be adding this space.

File: script/input/11155111/tokenVotingModuleConfig.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "comment": "This is an example token voting module deployment on Sepolia.",
-  "factory": "0x6A97643633eafEEC00b7Ec4CE84269203645aaBC",
+  "factory": "0xFBE17545dffD75A92A5A72926AE581478973FE65",
   "llamaCore": "0xc68046794327490F953EA15522367FFBA0b64f86",
```
Comment (AustinGreen) [inline script/input/11155111/tokenVotingModuleConfig.json:4]: Maybe we should make this the 0 address so it forces the user to provide their own llama core?

File: script/input/11155111/tokenVotingModuleConfig.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-  "tokenAdapterLogic": "0x8CCe1b824EfF3A7966348528B6951A84eC0541A5",
+  "tokenAdapterLogic": "0x088C268cb00226D6A9b29e5488905Aa94D2f0239",
   "tokenAddress": "0xf44d44a54440F22e5DC5adb7efA3233645f04007",
```
Comment (AustinGreen) [inline script/input/11155111/tokenVotingModuleConfig.json:6]: Same here

File: script/input/11155111/tokenVotingModuleConfig.json (hunk 2 above)
Comment (0xrajath) [reply script/input/11155111/tokenVotingModuleConfig.json:4]: Fair enough

File: script/input/11155111/tokenVotingModuleConfig.json (hunk 3 above)
Comment (0xrajath) [reply script/input/11155111/tokenVotingModuleConfig.json:6]: Fair enough

Comment (0xrajath) [reply script/input/11155111/tokenVotingModuleConfig.json]: Done in [44b1418](https://github.com/llamaxyz/llama-periphery/pull/95/commits/44b141866abe12800d1609c4d184144149c6f549)

File: script/input/11155111/tokenVotingModuleConfig.json (hunk 2 above)
Comment (0xrajath) [reply script/input/11155111/tokenVotingModuleConfig.json]: Done in [44b1418](https://github.com/llamaxyz/llama-periphery/pull/95/commits/44b141866abe12800d1609c4d184144149c6f549)

**Review by AustinGreen — APPROVED** [review]

**Pull request merged**